internal/cache/db_test.go

```diff
@@ -259,12 +268,14 @@ func TestUpdateFromUserInfo(t *testing.T) {
 		wantErr     bool
 	}{
 		// New user
-		"Insert new user":                 {userCase: "user1"},
-		"Update last login time for user": {userCase: "user1", dbFile: "one_user_and_group"},
+		"Insert new user":                              {userCase: "user1"},
+		"Update last login time for user":              {userCase: "user1", dbFile: "one_user_and_group"},
+		"Insert new user without optional gecos field": {userCase: "user1-without-gecos"},
```

Comment:
I know the ordering is not ideal, but `gofmt` was getting confused when I added the `"Insert new user without optional gecos field"` right below `"Insert new user"`

Reply:
sounds good to me :)